symfony/phpunit-bridge/4.1/bin/phpunit

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env php
+<?php
+
+if (!file_exists(dirname(__DIR__).'/vendor/symfony/phpunit-bridge/bin/simple-phpunit')) {
+    echo "Unable to find the `simple-phpunit` script in `vendor/symfony/phpunit-bridge/bin/`.\n";
+    exit(1);
+}
+if (false === getenv('SYMFONY_PHPUNIT_REMOVE')) {
+    putenv('SYMFONY_PHPUNIT_REMOVE=');
```

Comment:
is this actually needed ?

Reply:
yes: this sets the env vars to the empty string, so that the default behavior is bypassed (when getenv returns false, yaml+prophecy are removed otherwise)

Reply:
(and symfony/yaml is no longer a phpunit dep in recent version btw if one is wondering why we don't need this anymore)

Reply:
there was a suggestion in symfony to change the default list of removed package to not remove prophecy (matching what this script would do by default). It may be worth reconsidering it for 4.2 if we decide that the recipe should not remove anything.